Bump tar 1.30

stalchuk opened the pull request:
Bump:
tar -> 1.30
gzip -> 1.9
xz-utils -> 5.2.4

alllexx88 commented:
Thank you for your contribution. Some notes:

- `tar` build does not depend on `bzip2`, `gzip` and `xz-utils`, so we don't need to stage them when building `tar`. The binaries from `bzip2`, `gzip` and `xz-utils` packages are invoked by `tar` at runtime instead.
- Those `share/info/dir` remove commands where there to avoid situations where different packages had this file, thus needing `-force-overwrite` install flag to co-exist. We didn't remove `man` pages and docs before. I don't see much point in removing them: the increase in size is neglectable I think.
- I don't see why you removed `--disable-nls` configure flag for `gzip`. Could you explain this change?

stalchuk commented:
>     * `tar` build does not depend on `bzip2`, `gzip` and `xz-utils`, so we don't need to stage them when building `tar`. The binaries from `bzip2`, `gzip` and `xz-utils` packages are invoked by `tar` at runtime instead.

My bad. Fixed;
 
>     * Those `share/info/dir` remove commands where there to avoid situations where different packages had this file, thus needing `-force-overwrite` install flag to co-exist. We didn't remove `man` pages and docs before. I don't see much point in removing them: the increase in size is neglectable I think.

Following the conventions... :) Reverted;

>     * I don't see why you removed `--disable-nls` configure flag for `gzip`. Could you explain this change?

`--disable-nls` is unknown option for gzip configuration script, however, it is harmless... Reverted;
